client/src/components/Tool/ToolForm.vue

```diff
@@ -332,9 +323,16 @@ export default {
                     const nJobs = jobResponse && jobResponse.jobs ? jobResponse.jobs.length : 0;
                     if (nJobs > 0) {
                         this.showForm = false;
-                        this.showSuccess = true;
                         this.jobDef = jobDef;
                         this.jobResponse = jobResponse;
+                        const refJobId = this.jobResponse.jobs[0].id;
+                        const response = {
+                            jobDef: this.jobDef,
+                            jobResponse: this.jobResponse,
+                            toolName: this.toolName,
+                        };
+                        this.saveResponseForJobId(refJobId, response);
+                        this.$router.push(`/jobs/${refJobId}/success`);
```

Comment:
That route is misleading, you could read it as the job finished successfully, and a single jobId also doesn't really fit since you get an array of jobs. 

```suggestion
                        this.$router.push(`/jobs/submission/success`);
```
?

Reply:
> That route is misleading, you could read it as the job finished successfully

That makes sense, I'll change the route to `/jobs/submission/success`, but this will now be a temporary route instead of how I set it up: Now, if you refresh` /jobs/${refJobId}/success`, the `jobStore` is empty and hence, we reroute to `/jobs/${refJobId}/view`. I'll change it so we reroute to `/` instead
> , and a single jobId also doesn't really fit since you get an array of jobs.

So, the `jobDef` and `jobResponse` variables created in the `ToolForm` do not contain any other unique variable that could be used to store them in the `jobStore`, hence, I used `jobResponse.jobs[0].id`
But now, changing the route, I can instead make it so that we only store the most recent `ToolForm` run, instead of storing runs based on `jobResponse.jobs[0].id`